### PR TITLE
removing hardcoded urls with conf variables

### PR DESCRIPTION
**Pull request by mlautman**
### Description

This removes hardcoded URL's with conf.py variables 

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [x] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -38,32 +38,31 @@
...
 }
 
-# Experimental
-
-
-
 # Add any paths that contain custom themes here, relative to this directory.
 
 # Links
+ros_distro = 'melodic'
```
Comment (davetcoleman) [inline conf.py]: this should be cherry-picked to master and this string updated to master, also.

File: doc/benchmarking/benchmarking_tutorial.rst [hunk 2]
```diff
@@ -9,7 +12,7 @@ Getting Started
 ---------------
 If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
 
-The `benchmarking package <https://github.com/ros-planning/moveit/tree/melodic-devel/moveit_ros/benchmarks>`_ provides methods to benchmark motion planning algorithms and aggregate/plot statistics using the OMPL Planner Arena.
+The :moveit_codedir:`benchmarking package <moveit_ros/benchmarks>` provides methods to benchmark motion planning algorithms and aggregate/plot statistics using the OMPL Planner Arena.
```
Comment (davetcoleman) [inline doc/benchmarking/benchmarking_tutorial.rst]: so fancy!

File: conf.py (hunk 1 above)
Comment (mlautman) [reply conf.py]: Agreed. Now that I think about it, I think this should target master instead of melodic. I can cherry pick into melodic

**Comment by mlautman** [comment]
Cherry picked into melodic

**Pull request merged**